ext/http/http_next.rs

```diff
@@ -1299,6 +1300,23 @@ pub async fn op_http_wait(
       }
     }
 
+    if let HttpNextError::Other(err) = &err {
```

Comment:
Could you add a comment here explaining what's going on here?

Reply:
there's actually a comment above this diff explaining why we're filtering out specific errors here